tracking/ft/events/utils/fire-tracking.js

```diff
@@ -0,0 +1,10 @@
+export default (name, data, el = document.body) => {
```

Comment:
You can use https://github.com/Financial-Times/n-ui/blob/master/utils/index.js#L45 instead of defining this helper